src/flashbar/interfaces.ts

```diff
@@ -20,6 +20,9 @@ export namespace FlashbarProps {
     buttonText?: ButtonProps['children'];
     onButtonClick?: ButtonProps['onClick'];
     onDismiss?: ButtonProps['onClick'];
+    analyticsMetadata?: {
```

Comment:
You will need to add a `@analytics` to the tsdoc comment for items. You can see an example in [wizard](https://github.com/cloudscape-design/components/blob/e30984564d0899802df1265b4831228d46cea7bc/src/wizard/interfaces.ts#L40)